src/main/java/net/glowstone/io/entity/AreaEffectCloudStore.java

```diff
@@ -0,0 +1,30 @@
+package net.glowstone.io.entity;
+
+import net.glowstone.entity.GlowAreaEffectCloud;
+import net.glowstone.util.nbt.CompoundTag;
+import org.bukkit.Location;
+import org.bukkit.entity.EntityType;
+
+class AreaEffectCloudStore extends EntityStore<GlowAreaEffectCloud> {
+
+    public AreaEffectCloudStore() {
+        super(GlowAreaEffectCloud.class, EntityType.AREA_EFFECT_CLOUD);
+    }
+
+    @Override
+    public void load(GlowAreaEffectCloud entity, CompoundTag tag) {
+        super.load(entity, tag);
+        // TODO
```

Comment:
Can you add what NBT fields need to be added in the TODO?

Reply:
Done in the next commit.